docs/Transport/Measure-EmailDelayInMTL.md

```diff
@@ -0,0 +1,58 @@
+# Measure-EmailDelayInMTL
+Generates a report of the maximum message delay for all messages in an Message Tracking Log.
+
+## DESCRIPTION
+Gather message tracking log details of all message to / from a given recipient for a given time range. Useful for determining if a "slow" message was a one off or a pattern.
+
+Recommend using [Start-HistoricalSearch](https://learn.microsoft.com/en-us/powershell/module/exchange/start-historicalsearch?view=exchange-ps) in EXO to gather an detailed MTL for processing.
+
+``` PowerShell
+Start-HistoricalSearch -ReportTitle "Fabrikam Search" -StartDate 8/10/2024 -EndDate 8/12/2024 -ReportType MessageTraceDetail -SenderAddress michelle@fabrikam.com -NotifyAddress chris@contoso.com
+```
+
+## PARAMETER
+
+**-MTLFile**
+
+MTL File to process.
+
+**-ReportPath**
+
+Folder path for the output file.
```

Comment:
Let's use a similar format from SourceSideValidations.md. That way we are a little consistent with our work.

Reply:
Documentation updated.